bin/test-example

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
Does anyone know whether http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02 is a standard that can be relied upon by all POSIX-compliant shells, not just bash? If so, default variable is POSIX-compliant, which is all I need.

It is important to for me to be able to assume that `/bin/sh` isn't bash, but possibly `dash` - this fact might be true for at least one machine I own.